utils/tests/test_recipient_validation.py

```diff
@@ -97,7 +127,7 @@
     lambda number: number[0] not in {
         '712345678910',   # Could be Russia
     },
-    invalid_local_phone_numbers
+    invalid_international_phone_numbers
```

Comment:
seems like this changes things? is this a big deal?

Reply:
In few cases, we have different set of error messages for local and international phone numbers. 
And also our code is different from U.K's. So using the same set of error messages is failing. 
Hence i have to add new parameters(invalid_international_phone_numbers) for  international number validation. Please refer 'validate_phone_number_and_require_local'  & 'validate_phone_number_and_allow_international' methods. 

As per our code, when local number(prefixed with 0) exceeds 10 digits it is not a valid number. This is why i had to truncate many local phone numbers in testcase parameters.